events/api.py

```diff
@@ -586,8 +587,12 @@ def update(self, instance, validated_data):
                 raise PermissionDenied()
         else:
             # without api key, the user will have to be admin
-            if not instance.is_user_editable() or not instance.can_be_edited_by(self.user):
-                raise PermissionDenied()
+            if hasattr(instance, 'can_be_edited_by'):
```

Comment:
Actually, instead of disregarding `can_be_edited_by`, the method `can_be_edited_by` should be implemented in `models.py` for `Keyword` and `Place` instances.

Essentially, use the same method as in the `Event` model. They just use the `is_admin` method.

If you don't implement `can_be_edited_by`, the added `is_admin` methods in `Place` and `Keyword` are rather useless :)